src/Compilers/CSharp/Portable/Symbols/SignatureOnlyParameterSymbol.cs

```diff
@@ -104,10 +110,8 @@ public override int GetHashCode()
             return Hash.Combine(
                 _type.GetHashCode(),
                 Hash.Combine(
-                    Hash.CombineValues(_customModifiers),
```

Comment:
They don't have to contribute, that is the nature of the hash code. Especially given the fact that in the most common scenario there are no modifiers at all. This keeps the calculation faster and simpler.